lib/OpenAPI/Client.pm

```diff
@@ -37,8 +38,27 @@ has ua => sub { Mojo::UserAgent->new };
 sub call {
   my ($self, $op) = (shift, shift);
   my $code = $self->can($op);
-  return $self->$code(@_) if $code;
-  Carp::croak('[OpenAPI::Client] No such operationId');
+  Carp::croak('[OpenAPI::Client] No such operationId') if !$code;
+  $self->$code(@_);
```

Comment:
What does these two lines of diff do?

Reply:
Exactly the same, but more easily cargo-culted in `call_p`. I'm happy to drop this if you prefer.